src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -495,6 +495,9 @@ def _build_role_scope(resource_group_name, scope, subscription_id):
         if resource_group_name:
             err = 'Resource group "{}" is redundant because scope is supplied'
             raise CLIError(err.format(resource_group_name))
+        from msrestazure.tools import is_valid_resource_id
+        if scope.startswith('/subscriptions/') and not is_valid_resource_id(scope):
```

Comment:
Can `/` be used as scope?

Reply:
Good question. If the user specify scope as `/`, CLI will try to create a role assignment under the global resource provider `/providers/Microsoft.Authorization/roleAssignments/{uuid}` instead of `/subscriptions/{uuid}/providers/Microsoft.Authorization/roleAssignments/{uuid}`. Normally server will return error which tells "does not have authorization to perform ... or scope is invalid".
I'm wondering whether there are already some users who take this benefit to create role assignments under global resource provider. So I think it's better to keep this and rely on the service side to do the validation.